build/win64/install-debug.bat

```diff
@@ -1,4 +1,4 @@
-call "%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+call "D:\Program (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
```

Comment:
Can you revert the changes to this file?

Reply:
I'm guessing this was added because of my recommendation above. Something that would have worked would have been:
```sh
git add docs plugins
git commit
```
this would only add changes to the "docs" and "plugins" folder to the commit.